grafana/grafana
# Logs: move logs-components from grafana-ui to grafana-main

this pull request moves the logs-related components from `grafana-ui` to the main grafana package.

(to keep backward compoatibility, they also remain in `grafana-ui`, they will be deleted in grafana10)

this pull request is quite large, i recommend reading it commit by commit. the first commit is the largest, but that just copies the files unmodified to the new place. the remaining commits adjust the files, and those commits are much smaller.

what is not included in the PR:
1. mark the original files in grafana-ui as deprecated, will be done in a separate PR when this is merged
2. new betterer-exceptions had to be added, because these exceptions already existed for the `grafana-ui` version of the files. i tried to fix all these betterer-exceptions at the new place, but not all have an easy fix, some require changes also to files not in this PR, so it will be better to do those as a separate PR


how to test:
- no specific instructions, run it, open the logs-visualization in both explore and in dashboard, try it out, make sure there are no errors.
